app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max: string;
+  value: string;
+  children?: React.ReactNode;
+};
+
+type SliderStates = {
+  value: string;
```

Comment:
this one should be something like `currentValue` because it represents current state